### PR TITLE
update SQLAlchemy to version 1.3.3 and its dependencies

### PR DESCRIPTION
**Pull request by GiancarloFusiello**

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -225,39 +225,20 @@
         },
         "psycopg2": {
```
Comment (giorgiosironi) [inline Pipfile.lock]: Separate point, several projects has switched to `psycopg2-binary` which seems easier to install as it doesn't compile code, opinions? Not for the scope of this PR of course.

**Comment by giorgiosironi** [comment]
(waiting on test results however)

**Comment by giorgiosironi** [comment]
The failure looks related (`SQLAlchemy` or `psycopg`):
```
[2019-04-17T15:40:13.482Z]     return connector.get_engine()
[2019-04-17T15:40:13.482Z]   File "/srv/profiles/venv/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py", line 559, in get_engine
[2019-04-17T15:40:13.482Z]     self._engine = rv = sqlalchemy.create_engine(info, **options)
[2019-04-17T15:40:13.482Z]   File "/srv/profiles/venv/lib/python3.6/site-packages/sqlalchemy/engine/__init__.py", line 435, in create_engine
[2019-04-17T15:40:13.482Z]     return strategy.create(*args, **kwargs)
[2019-04-17T15:40:13.482Z]   File "/srv/profiles/venv/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 87, in create
[2019-04-17T15:40:13.482Z]     dbapi = dialect_cls.dbapi(**dbapi_args)
[2019-04-17T15:40:13.482Z]   File "/srv/profiles/venv/lib/python3.6/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 632, in dbapi
[2019-04-17T15:40:13.482Z]     import psycopg2
[2019-04-17T15:40:13.482Z]   File "/srv/profiles/venv/lib/python3.6/site-packages/psycopg2/__init__.py", line 50, in <module>
[2019-04-17T15:40:13.482Z]     from psycopg2._psycopg import (                     # noqa
[2019-04-17T15:40:13.482Z] ImportError: libpq.so.5: cannot open shared object file: No such file or directory
```

File: Pipfile.lock (hunk 1 above)
Comment (GiancarloFusiello) [reply Pipfile.lock]: more information about this can be found here: https://github.com/psycopg/psycopg2/issues/674

Comment (GiancarloFusiello) [reply Pipfile.lock]: I would recommend staying with the current dependency for this project.

File: Dockerfile [hunk 2]
```diff
@@ -9,7 +9,10 @@ WORKDIR ${PROJECT_FOLDER}
 USER root
 RUN mkdir -p var/logs && \
     chown --recursive elife:elife . && \
-    chown www-data:www-data var/logs
+    chown www-data:www-data var/logs && \
+    apt-get update && \
+    apt-get install -yqq --no-install-recommends \
```
Comment (giorgiosironi) [inline Dockerfile]: can usually do an `apt-get clean` in the same `RUN`

**Pull request merged**